src/test/java/org/minions/devfund/angela/battleship/ShipTest.java

```diff
@@ -0,0 +1,23 @@
+package org.minions.devfund.angela.battleship;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test for {@link Ship}.
+ */
+public class ShipTest {
+
+    private Ship ship;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "ship" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)